http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzureTestUtils.scala

```diff
@@ -51,6 +57,21 @@ object AzureTestUtils extends MockitoSugar {
         .errorReport(new ErrorReport())
     }
 
+    // Create storage container
+    when {
+      api.createAzureStorageContainer(any, ArgumentMatchers.eq(workspaceId.value))
+    } thenAnswer { invocation =>
+      val requestBody = invocation.getArgument[CreateControlledAzureStorageContainerRequestBody](0)
+      new CreatedControlledAzureStorageContainer().resourceId(UUID.randomUUID())
+    }
+
+    // Create storage container
+    when {
+      api.createAzureStorageContainer(any, ArgumentMatchers.eq(workspaceId2.value))
+    } thenAnswer { _ =>
+      throw new Exception("test exception")
```

Comment:
probably worth having an argument to the function here to indicate whether we want this call to error with the default case being it not throwing an error. I doubt we want all tests that leverage these mocks to have an error in `createAzureStorageContainer`

Reply:
yeah I had it only trigger for workspace2, but that's a non-intuitive change, so I can alter that